src/Adapter/MSTest.TestAdapter/TestingPlatformAdapter/TestingPlatformBuilderHook.cs

```diff
@@ -11,6 +11,17 @@ namespace Microsoft.VisualStudio.TestTools.UnitTesting;
 public static class TestingPlatformBuilderHook
 {
 #pragma warning disable IDE0060 // Remove unused parameter
-    public static void AddExtensions(ITestApplicationBuilder testApplicationBuilder, string[] arguments) => testApplicationBuilder.AddMSTest(() => [Assembly.GetEntryAssembly()!]);
+    public static void AddExtensions(ITestApplicationBuilder testApplicationBuilder, string[] arguments)
+    {
+#if NET8_0_OR_GREATER
```

Comment:
Could be `NETCOREAPP` or `NET7_0_OR_GREATER` if we want to link it specifically to native aot.

Reply:
IIRC it did not build with just netcoreapp, and net7 is out of life and rest of the platform used net8 so I used net8